Doc/library/colorsys.rst

```diff
@@ -37,6 +38,20 @@ The :mod:`colorsys` module defines the following functions:
    Convert the color from YIQ coordinates to RGB coordinates.
 
 
+.. function:: rgb_to_yuv(r, g, b)
+
+   Convert the color from RGB coordinates to YUV coordinates.
```

Comment:
I suggest to mention "ATSC BT.709 constants" in the doc. Same for yuv_to_rgb().

Reply:
I've updated the YIQ functions to mention the FCC NTSC constants as well.